SublimeText/README.md

```diff
@@ -8,8 +8,8 @@ Go ahead and [download](http://www.sublimetext.com/) it. Open the **.dmg** file,
 
 Sublime Text is not free, but it has an unlimited "evaluation period". Anyhow, we're going to be using it so much that even the seemingly expensive $60 price tag is worth every penny. If you can afford it, It is adviced that you [support](http://www.sublimetext.com/buy) this awesome tool. :)
 
-Let's create a shortcut so we can launch Sublime Text from the command-line:
+ZSH has a plugin to create shortcuts to launch Sublime Text from the command-line. Simply open your `~/.zshrc` file and make sure that the  `sublime` plugin is part of your plugins:
```

Comment:
I think this is oh-my-zsh specific as prezto doesn't seem to have this plugin based on the comments in the PR https://github.com/sorin-ionescu/prezto/pull/671. 

Let's improve the documentation on creating the shortcut instead as that is shell agnostic as well. Thoughts?